src/service/mustwatch_service.py

```diff
@@ -0,0 +1,304 @@
+                            chat_id: str,
+                            chosen_user_id: str,
+                            user_score: int) -> None:
+        users_id = chosen_user_id[0]
+        watch_record = self.watches_read.get_record_by_title_and_chat_id(title, chat_id)
+        self.mustwatches_redact.update_user_score(users_id, watch_record, user_score)
+
+    def __delete_mustwatches_and_watches(self,
+                                         title: str,
+                                         chat_id: str) -> None:
+        watches_id_list = self.watches_read.get_watches_id_list_with_same_title(title, chat_id)
+        self.mustwatches_redact.delete_mustwatch(watches_id_list)
+        self.watches_redact.delete_watch(watches_id_list)
+
+    def __get_user_request(self,
+                           chat_id: str,
+                           user_id: str) -> Query:
+        user = self.users_read.get_record(chat_id, user_id)
+        return self.user_requests_read.get_record(user)
+
+    def __is_title_filled_at_user_request(self,
+                                          chat_id: str,
+                                          user_id: str) -> bool:
+        user_request = self.__get_user_request(chat_id, user_id)
+        return bool(user_request.title)
+
+    def __get_add_or_delete(self,
+                            chat_id: str,
+                            user_id: str) -> bool:
+        user_request = self.__get_user_request(chat_id, user_id)
+        return user_request.add_or_delete
+
+    def prepare_user_request(self,
+                             chat_id: str,
+                             user_id: str,
+                             message_id: str) -> None:
+        try:
+            user = self.users_read.get_record(chat_id, user_id)
+            self.user_requests_redact.update_message_id(user, message_id)
+            self.user_requests_redact.update_title(user, RegisterService.FILL_DATA)
+            self.user_requests_redact.delete_user_score(user)
+            self.session.commit()
+        except:
+            self.session.rollback()
+
+    def is_callback_protected_from_intruder(self,
+                                            chat_id: str,
+                                            user_id: str,
+                                            message_id: str) -> bool:
+        user = self.users_read.get_record(chat_id, user_id)
+        is_same_user_and_message = bool(self.user_requests_read.get_record_by_users_and_message_ids(user, message_id))
+        return is_same_user_and_message
+
+    def update_user_request_title_transaction(self,
+                                              chat_id: str,
+                                              user_id: str,
+                                              title: str) -> None:
+        try:
+            user = self.users_read.get_record(chat_id, user_id)
+            self.user_requests_redact.update_title(user, title)
+            self.session.commit()
```

Comment:
By the way, you could create your own context manager for it and transorm this code into `with as` statement